src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScore.java

```diff
@@ -113,30 +113,30 @@ static VersionStats calculateVersionStats(Collection<VersionInfo> versionInfos)
   /**
    * Create the version info based on list of sorted artifact versions.
    *
-   * @param artifactVersions sorted collection of artifact versions
-   * @return version infos
+   * @param artifactVersions A sorted collection of artifact versions.
+   * @return Version info.
```

Comment:
A collection of `VersionInfo`?

Reply:
Or did you mean to say that `VersionInfo` is itself a `List`?

Reply:
I mean information (info) about versions. If I understand correctly, we can't say "informations" or "infos"

https://dictionary.cambridge.org/grammar/british-grammar/information

Reply:
How about `VersionsInfo`? Just a way to indicate that it is a `Collection` of some sort.

Reply:
I updated it to "A collection of {@link VersionInfo}."

I don't think that is strictly necessary to tell what kind of class is returned by a method. I think it is more important to describe what kind of data it returns. In this case, the method returns information about artifacts. That's why it think "Version info" was enough.